app/build.gradle

```
@@ -41,6 +41,11 @@ android {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            buildConfigField "android.net.Uri", "OAUTH_SERVER", "android.net.Uri.parse(\"https://api.traewelldroid.de\")"
```

```suggestion
            buildConfigField "android.net.Uri", "OAUTH_SERVER", "android.net.Uri.parse(\"https://app.traewelldroid.de\")"
```
I think that should be the callback url?

---

I am still not fully convinced that it is a great Idea to have the server run on the deeplink URL, but ig it works